test_pool/io_virt/test_i007.c

```diff
@@ -39,11 +39,9 @@ static void payload()
 
   while (num_smmu--) {
       if (val_smmu_get_info(SMMU_CTRL_ARCH_MAJOR_REV, num_smmu) == 2) {
-          if (g_sbsa_level > 3) {
```

Comment:
For level equal to 3, SMMU v3 support is not mandatory. It is required only from level 4 and above. So, this change is not required.